packages/sdk-core/src/modules/database/BacktraceDatabase.ts

```diff
@@ -41,7 +46,10 @@ export class BacktraceDatabase implements BacktraceModule {
         private readonly _sessionFiles?: SessionFiles,
     ) {
         this._databaseRecordContext = new BacktraceDatabaseContext(this._options?.maximumRetries);
-        this._maximumRecords = this._options?.maximumNumberOfRecords ?? 8;
+        this._recordLimits = {
+            report: this._options?.maximumNumberOfRecords ?? 8,
+            attachment: this._options?.maximumNumberOfAttachmentRecords ?? 100,
```

Comment:
one more idea - 100 is a random number and based on this we might have some problems. Maybe we should just agree on 5 attachments per session - 30? Usually, users want to send 1-2 attachments + 1-2 for breadcrumbs, so 5/session is a safe value

Reply:
30 is as much arbitrary as 100 is. Default maximum number of previous sessions is 1, though this could be changed, and not necessarily available. I'll change this to 10, as it seems more reasonable. This doesn't cost us disk space, as each attachment record is little in size.